dashboard/app/controllers/application_controller.rb

```diff
@@ -325,6 +328,28 @@ def clear_sign_up_session_vars
     end
   end
 
+  # Check that the user is compliant with the Child Account Policy. If they
+  # are not compliant, then we need to send them to the lockout page.
+  def assert_child_account_policy
+    # Check that the child account policy is currently enabled.
+    return unless CPA.cpa_experience(request)
+
+    # Anonymous users are NOT affected by our Child Account Policy
+    return unless current_user
+
+    # URLs we should not redirect.
+    return if Set[
+      # Don't block any user from signing out
+      '/users/sign_out',
+      # Don't block any user from changing the language
+      '/locale',
+      # Avoid an infinite redirect loop to the lockout page
+      '/lockout',
+    ].include?(request.path)
+
+    redirect_to '/lockout' unless current_user.child_account_policy_compliant?
```

Comment:
Please don't use hardcoded rails paths
```suggestion
      destroy_user_session_path
      # Don't block any user from changing the language
      locale_path,
      # Avoid an infinite redirect loop to the lockout page
      lockout_path,
    ].include?(request.path)

    redirect_to lockout_path unless current_user.child_account_policy_compliant?
```

Reply:
Oh wow, I didn't know the routes.rb did this automatically!